emails/migrations/0062_move_profile_and_registered_subdomain_models.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 4.2.13 on 2024-06-07 22:08
+# Move Profile and RegisteredSubdomain to privaterelay app
+# Other half is
+# privaterelay/migrations/0010_move_profile_and_registered_subdomain_models.py
+# See https://davit.hashnode.dev/django-move-model
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("emails", "0061_relayaddress_idx_ra_created_by_addon"),
+    ]
+
+    state_operations = [
+        migrations.RemoveField(model_name="profile", name="user"),
+        migrations.DeleteModel(name="RegisteredSubdomain"),
+        migrations.DeleteModel(name="Profile"),
+    ]
```

Comment:
(praise) No `migrations.AlterField` operation needed since no models directly uses `Profile` and instead uses `User` 🤯 
(question, blocking) Why is the [RemoveField](https://docs.djangoproject.com/en/5.0/ref/migration-operations/#removefield) operation there? Wouldn't this result in dropping the `user` column in the `emails_profile` table?

Reply:
Thanks @jwhitlock for this write up. Here's my understood summary: 
because we separated the model state and the database using  `state_operations(migrations.SeparateDatabaseAndState(state_operations=state_operations))` the `RemoveField` will only change the model state and will not result in DB changes (which I can confirm when running the migration locally).

Reply:
yep, you've got it!